tests/ui_tests/app/page/common/base_page.py

```diff
@@ -468,6 +463,19 @@ def get_in_progress_job_amount_from_header(self):
         self.wait_element_visible(AuthorizedHeaderLocators.job_popup)
         return self.find_element(AuthorizedHeaderLocators.JobPopup.in_progress_jobs).text.split("\n")[1]
 
+    def get_failed_job_amount_from_header(self):
```

Comment:
why it is part of `BasePageObject`? we have a `PageHeader` object

Reply:
it is method of class PageHeader(BasePageObject)

Reply:
:fearful: